src/main/java/com/solab/iso8583/IsoType.java

```diff
@@ -73,6 +73,11 @@ public enum IsoType {
 	 * Similar to {@link IsoType#ALPHA} but holds byte arrays instead of strings.
 	 */
 	BINARY(true, 0),
+    /**
```

Comment:
Indentation is different from the rest

Reply:
I'll reformat everything in a different mr - looks like the project in general doesn't follow default formatting.